setup.py

```diff
@@ -1,8 +1,14 @@
 from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
 setup(
-    name='scp_ingest_pipeline',
-    version='0.1',
+    name='scp-ingest-pipe',
+    version='1.3.6rc1',
```

Comment:
The repo is currently on release 1.3.6
The state of master has code that will be in release 1.3.7 - should our version be 1.3.6rc1 or 1.3.7rc1?

Reply:
`1.3.7rc1`, I think.

Also, `scp-ingest-pipeline` instead of `scp-ingest-pipe`.

Reply:
Thanks for catching scp-ingest-pipe! Fixed and using 1.3.7rc1 in `9a25885`